Create service_eval.md

schanduae4 opened the pull request.

schanduae4 committed:
Create service_eval.md

kungfuchicken reviewed (changes requested):
Please, cite any sources used to generate this evaluation. This does not need to be an academic citation. Links to information on the web is sufficient. Just ensure you cite specific pages, and not just the organization's home page.

This evaluation does not mention the Theory of Change framework.

Please, make sure you are following the structure identified in the original requirements.

schanduae4 committed:
Update service_eval.md

Updated from perspective of theory of change, as well servant leadership along with artciles published by SLSC reference

schanduae4 committed:
Update service_eval.md

updating with theory of change methodologies evaluation and servant as a leader, also references of articles published by slsc about their events, engagements etc.
